gqlalchemy/exceptions.py

```diff
@@ -109,13 +111,33 @@ def __init__(self):
         self.message = ORDER_BY_TYPE_ERROR
 
 
-class GQLAlchemyLiteralAndExpressionMissingInWhere(GQLAlchemyError):
-    def __init__(self):
+class GQLAlchemyLiteralAndExpressionMissingInClause(GQLAlchemyError):
+    def __init__(self, clause: str):
         super().__init__()
-        self.message = LITERAL_AND_EXPRESSION_MISSING_IN_WHERE
+        self.message = LITERAL_AND_EXPRESSION_MISSING.format(clause=clause)
 
 
-class GQLAlchemyExtraKeywordArgumentsInWhere(GQLAlchemyError):
+class GQLAlchemyLiteralAndExpressionMissingInWhere(GQLAlchemyLiteralAndExpressionMissingInClause):
     def __init__(self):
+        super().__init__(clause="WHERE")
```

Comment:
does it make sense exposing these keywords to a separate enum? and then importing it. for example

class QueryConstants(Enum):
WHERE = "WHERE"

Reply:
and so on...

Reply:
Oh, you meant in exceptions :D Sure, it makes sense to me. We have something similar in the query_builder, but to avoid circular imports, I'll create a new enum in exceptions and you can let me know what you think.

Reply:
@Josipmrden added, please review